api/src/pcapi/core/finance/repository.py

```diff
@@ -452,13 +457,188 @@ def _get_reimbursement_details_from_invoices_base_query(invoice_ids: list[int])
+            educational_models.CollectiveOffer.name.label("offer_name"),
+            offerers_models.Venue.name.label("venue_name"),
+            offerers_models.Venue.common_name.label("venue_common_name"),  # type: ignore[attr-defined]
+            # Sometimes, a venue has a postal code and a city, but no address, and the offerer's address
+            # is in another city. Now, we only check the postal code to keep either the venue's full address
+            # or the offerer's one
+            sqla.case(
+                (offerers_models.Venue.postalCode.is_not(None), offerers_models.Venue.address),
+                else_=offerers_models.Offerer.address,
+            ).label("venue_address"),
+            sqla.case(
+                (offerers_models.Venue.postalCode.is_not(None), offerers_models.Venue.postalCode),
+                else_=offerers_models.Offerer.postalCode,
+            ).label("venue_postal_code"),
+            sqla.case(
+                (offerers_models.Venue.postalCode.is_not(None), offerers_models.Venue.city),
+                else_=offerers_models.Offerer.city,
+            ).label("venue_city"),
+            offerers_models.Venue.siret.label("venue_siret"),
+            offerers_models.Venue.departementCode.label("venue_departement_code"),
+            # See note about `amount` in `core/finance/models.py`.
+            (-models.Pricing.amount).label("amount"),
+            models.Pricing.standardRule.label("rule_name"),
+            models.Pricing.customRuleId.label("rule_id"),
+            models.Pricing.collectiveBookingId.label("collective_booking_id"),
+            sqla.cast(models.Invoice.date, sqla.Date).label("invoice_date"),
+            models.Invoice.reference.label("invoice_reference"),
+            models.CashflowBatch.cutoff.label("cashflow_batch_cutoff"),
+            models.CashflowBatch.label.label("cashflow_batch_label"),
+            models.BankAccount.label.label("bank_account_label"),
+            models.BankAccount.iban.label("iban"),
+            sqla.case((models.FinanceEvent.bookingFinanceIncidentId.is_(None), False), else_=True).label("is_incident"),
+        )
+    ).all()
+
+
+def _get_collective_reimbursement_details_from_invoices(invoice_ids: list[int]) -> list[tuple]:
+    collective_details = _get_collective_booking_reimbursement_data(
+        _get_reimbursement_details_from_invoices_base_query(invoice_ids).join(
+            educational_models.CollectiveBooking, models.Pricing.collectiveBooking
+        )
+    )
+    collective_details.extend(
+        _get_collective_booking_reimbursement_data(
+            _get_reimbursement_details_from_invoices_base_query(invoice_ids)
+            .join(models.Pricing.event)
+            .join(models.FinanceEvent.bookingFinanceIncident)
+            .join(models.BookingFinanceIncident.collectiveBooking)
+        )
+    )
+
+    return collective_details
+
+
+def _get_individual_booking_reimbursement_data(query: BaseQuery) -> list:
```

Comment:
Détail (encore), c'est surtout par cohérente avec la fonction appelante
```suggestion
def _get_individual_booking_reimbursement_data(query: BaseQuery) -> list[tuple]:
```